trio/_core/_asyncgens.py

```diff
@@ -174,10 +183,12 @@ async def finalize_remaining(self, runner):
             for agen in batch:
                 await self._finalize_one(agen, name_asyncgen(agen))
 
-    def close(self):
+    def close(self) -> None:
         sys.set_asyncgen_hooks(*self.prev_hooks)
 
-    async def _finalize_one(self, agen, name):
+    async def _finalize_one(
+        self, agen: AsyncGenerator[Any, Any], name: str | object
```

Comment:
`name` here could just be `object`, the `str` is redundant. It really doesn't have much to do with strings, the logging call below just does `repr()`.

Reply:
Changed in 0205c8f